main/CTLIterator.ml

```diff
@@ -248,14 +182,20 @@ module CTLIterator(D: RANKING_FUNCTION) = struct
     in InvMap.iter (fun l a -> Format.fprintf fmt "%a:\n%a\nDOT: %a\n" label_print l D.print a D.print_graphviz_dot a) inv
```

Comment:
Can you make the printing of the 'dot' format optional? If one runs the command-line version of the analysis, the output is polluted by this.